funnel/templates/email_project_registration.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "email_layout.html.jinja2" %}
+{% block content -%}
+
+<p>{% trans %}Hi{% endtrans %} {{ user.fullname }},</p>
```

Comment:
`{% trans user=user.fullname %}Hi {{user}},{% endtrans %}`

This will allow translations where the "Hi" can appear after the name.